packages/react-scripts/package.json

```diff
@@ -58,7 +58,7 @@
     "json-loader": "0.5.4",
     "object-assign": "4.1.0",
     "opn": "4.0.2",
-    "path-exists": "3.0.0",
+    "path-exists": "2.1.0",
```

Comment:
Yeah, I think it's fine to use the same old version here for consistency. The new version doesn't have any additional functionality.